index.html

```diff
@@ -16,8 +16,9 @@
             <input type="text" name="lastname" placeholder="Create a password"><br>
         </form>
         <p>Make sure it's more than 15 characters, or at least 8 characters, including a number, and a lowercase letter.</p>
-        <button id="loginButton">Sign up for GitHub</button>
+        <button id="loginButton">Sign up for GitHub</button> 
         <p id="footer">By clicking "Sign up for GitHub" you agree to our <a>terms of service</a> and <a>privacy statement</a>. We'll occasionally send you account related emails.</p>
+        
```

Comment:
https://github.com/jkoprcina/GitHubLoginScreen/blob/9d7fc05ad6f75d49cc5b317fee6986c9edb78891/main.css#L16-L17
Again, instead of percentage, try writing `padding: 10px;` and `margin: 5px 0 15px 0;`